build/db_schemas/scd/upto-v3.2.0-add_ovn_columns.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE scd_operations
+    ADD COLUMN IF NOT EXISTS uss_requested_ovn STRING
+        CHECK (uss_requested_ovn != ''), -- uss_requested_ovn must be NULL if unspecified, not an empty string
+    ADD COLUMN IF NOT EXISTS past_ovns         STRING[] NOT NULL
+        DEFAULT ARRAY []::STRING[]
+        CHECK (
+            array_position(past_ovns, NULL) IS NULL AND
+            array_position(past_ovns, '') IS NULL AND
+            array_position(past_ovns, uss_requested_ovn) IS NULL
+            ); -- past_ovns must not contain NULL elements, empty strings or current uss_requested_ovn
```

Comment:
@BenjaminPelletier @barroco FTR: the only modification this data migration is doing is to add columns. Given that the queries are only ever referencing explicitly columns (as opposed to a wildcard selection), I think it is safe to conclude that previous versions of the DSS runtime are compatible with this newer version of the schema.

(follow-up to the discussion last tuesday about upgrades during the InterUSS weekly)